srtcore/srt.h

```diff
@@ -175,12 +175,14 @@ typedef enum SRT_SOCKOPT {
    SRTO_CONGESTION,          // Congestion controller type selection
    SRTO_MESSAGEAPI,          // In File mode, use message API (portions of data with boundaries)
    SRTO_PAYLOADSIZE,         // Maximum payload size sent in one UDP packet (0 if unlimited)
-   SRTO_TRANSTYPE,           // Transmission type (set of options required for given transmission type)
+   SRTO_TRANSTYPE = 50,      // Transmission type (set of options required for given transmission type)
```

Comment:
Please don't change existing values. This breaks API.

Reply:
I don't. I'm simply declaring it explicitly for clarity.

Reply:
gotcha